tests/outputs/stmtTests.mg.out

```diff
@@ -1 +1,15 @@
 tests/inputs/stmtTests.mg:18:9: Error: function not in scope in tests.inputs.stmtTests.StmtTests: _=_ with arguments of type (T1, T2)
+
+tests/inputs/stmtTests.mg:33:9: Type error in tests.inputs.stmtTests.VarExprTests: got conflicting type annotation for var x: x has type T but type annotation is U
+
+tests/inputs/stmtTests.mg:56:29: Type error in tests.inputs.stmtTests.CallExprTests: could not deduce return type of call to overload_return_type. Possible candidates have return types [T, U]. Consider adding a type annotation
+
+tests/inputs/stmtTests.mg:68:9: Error in tests.inputs.stmtTests.LetExprTests: variable already defined in scope: x
+
+tests/inputs/stmtTests.mg:75:9: Not implemented in tests.inputs.stmtTests.LetExprTests: can not yet declare a variable without specifying its type or an assignment expression
```

Comment:
Indeed, I see that this is left as an error in the compiler. Perhaps we should change this error to something that explicitly forbids it (and maybe later relax that rule if we decide to do that).

Reply:
Yes I can look in to it.